ruby/ql/lib/codeql/ruby/dataflow/internal/DataFlowDispatch.qll

```diff
@@ -318,6 +316,19 @@ private predicate extendCallModule(Module m, Module n) {
   )
 }
 
+/**
+ * Gets a method available in module `m`, or in one of `m`'s transitive
+ * sub classes when `exact = false`.
+ */
+pragma[nomagic]
+private Method lookupMethod(Module m, string name, boolean exact) {
+  result = lookupMethod(m, name) and
+  exact in [false, true]
+  or
+  result = lookupMethodInSubClasses(m, name) and
```

Comment:
This change looks like it would make https://github.com/github/codeql/pull/10747 mostly obsolete, right ?

Reply:
It still only considers sub classes, not all ancestors.